fed/mosaic-cell/src/main/java/org/eclipse/mosaic/fed/cell/ambassador/CellAmbassador.java

```diff
@@ -237,30 +237,39 @@ protected void processInteraction(Interaction interaction) throws InternalFedera
      */
     private void process(CellularCommunicationConfiguration interaction) throws InternalFederateException {
         Validate.notNull(interaction.getConfiguration(), "CellConfiguration is null");
-        final String nodeId = interaction.getConfiguration().getNodeId();
+        long interactionTime = interaction.getTime();
+        CellConfiguration cellConfiguration = interaction.getConfiguration();
+        if (cellConfiguration.getMaxDownlinkBitrate() == null || cellConfiguration.getMaxUplinkBitrate() == null) {
+            cellConfiguration.setBitrates(
+                    ConfigurationData.INSTANCE.getNetworkConfig().defaultDownlinkCapacity,
+                    ConfigurationData.INSTANCE.getNetworkConfig().defaultUplinkCapacity
```

Comment:
ok, I guess this answerts my question above. 😁

Reply:
However, my downlink capacity would be overwritten if I would set downlink capacity only, right? 

Maybe this would be better:
```
cellConfiguration.setBitrates(
    ObjectUtils.defaultIfNull(cellConfiguration.getMaxDownlinkBitrate(), ConfigurationData.INSTANCE.getNetworkConfig().defaultDownlinkCapacity),
    ObjectUtils.defaultIfNull(cellConfiguration.getMaxUplinkBitrate(), ConfigurationData.INSTANCE.getNetworkConfig().defaultUplinkCapacity)
);
```

Reply:
Depends, do we want users to be able to only set one of the bitrates? I would argue, that we should limit users to only set both bitrates at the same time. Following the logic of either I care about the bitrates enough to manually set them both or I don't care about the bitrates at all,